internal/gensupport/send.go

```diff
@@ -43,7 +64,7 @@ func send(ctx context.Context, client *http.Client, req *http.Request) (*http.Re
 	if err != nil {
 		select {
 		case <-ctx.Done():
-			err = ctx.Err()
+			err = wrappedCallErr{ctx.Err(), err}
```

Comment:
Let's not make this change in `send()` for now. Reasons:

1. The case that the call returned a non-context error but the context is expired/canceled is not a very common case without retries being in the picture. And the string from `wrappedCallErr` is specific to retries being at the root of the issue.
2. `send` is much more widely used than `sendAndRetry` including by the manual clients built on top of this library. I would want to audit these to make sure they can handle this error (meaning that they are upgraded to use `errors.Is` rather than checking for equality) if they have error handling logic at that layer.

Reply:
fixed!